app/models/container_service.rb

```diff
@@ -6,7 +6,7 @@ class ContainerService < ApplicationRecord
 
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
   has_and_belongs_to_many :container_groups, :join_table => :container_groups_container_services
-  has_many :container_routes
+  has_many :container_routes, :dependent => :destroy
```

Comment:
Is this one correct @agrare ?  projects has many routes and services has many routes?  Should projects have routes through services?  Or maybe there can be routes not attached to a service?  🤷

Reply:
Are we leaving routes orphaned @jrafanie ?

A container route is its own "top-level" managed object so we would destroy it when we get the destroy event from k8s (versus e.g. a container which is only part of a container_group and wouldn't get its own destroy event)  I'm surprised it isn't at least dependent nullify though.

Reply:
No.  I'm seeing less than 200 routes where container conditions, env vars, volumes, security contexts, port configs, and custom attributes are the big ones in the container area over 1 million rows.

I can change it to `nullify`.  Are there others here that should be treated in the same way?  I don't want to change behavior.  I'll verify with my table counts from various databases.